### DISCUSSION TITLE
Upgrade caused db error for basic login

### DISCUSSION THREAD

**Discussion by gpjorgenson** [Budibase/budibase#5808] [Help] [answered]
Since upgrading our locally hosted budibase to version 1.0.148 from 1.0.90, signing in as a basic user that can access to only 2 apps, clicking on an app returns the error: {"message":"Missing/invalid DB name","status":500} ...

It works fine in the admin/developer preview mode with basic selected. It was working fine before the upgrade. Surprisingly, reversion did not help. There is no db 'status' ... There was an options field named 'status' in 2 of the five tables in the app's db but I removed those fields from the tables and the app to no avail.

Right after the upgrade, I had successfully demoed the app in the admin's Preview mode. I went to sign in with an user who only had basic level access and received the error.  I have tried too many things to delineate all of them. Has anyone else had this issue? Any solutions?

**Comment by shogunpurple** [comment]
Hi @gpjorgenson - could you try on the latest version? I'm not able to recreate this one unfortunately. You could try importing/exporting your app and seeing if that potentially makes a difference.

If it persists, it might help to see your app export in an email to support@budibase.com to help us recreate.

**Reply by gpjorgenson** [reply]
Thank you for the quick reply and I apologize for my delay. Due to some other responsibilities, I had to set this aside and have everyone use the old method until i can resolve this issue.
Anyway, I did as you suggested and did an app export and import nut it had no effect on the error:
![bd-invalid-db-error](https://user-images.githubusercontent.com/23404511/169097152-4bbd91ce-77ab-4f27-a380-4d4de7630745.png)

**Reply by gpjorgenson** [reply]
Note that this is when a limited access user, in this case having 'basic' level access only, tries to access the app.

**Reply by gpjorgenson** [reply]
I have been trying different things. I changed the limited user to have admin and developer permissions. Then went in and tried to view app. It still received the error. I then went into the 'Edit' and published the app user the user that I had just upgraded the permissions for. I then went out and did a 'View App' and it worked. That is until I logged that user out and went to my normal user. I did a 'View App' and received the error. Any users gets the error then. I went in and did another publish and for the user who published the app, it worked. Until I switched users again and then I got the error again. After that it worked for no users until I did another publish where it worked only for he user that published it. 
Here are the additional error info the 'Review App' Provides that the limited user did not receive.
![bd-invalid-db-error-viewapp](https://user-images.githubusercontent.com/23404511/169170825-9f58c8c6-781f-41e5-8128-9d3f08917083.png)
![bd-invalid-db-error-headers](https://user-images.githubusercontent.com/23404511/169170837-3d5a85c3-5f5e-40c4-9c72-6115b1f4fee8.png)

**Comment by gpjorgenson** [comment, marked as answer]
A complete update of budibase and docker by my programmer seems to have fixed this issue.
